pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -383,6 +387,68 @@ func ensureKubeAPIServerTrustedCA(client coreclientv1.CoreV1Interface, recorder
 	return err
 }
 
+func ensureLocalhostRecoverySAToken(client coreclientv1.CoreV1Interface, recorder events.Recorder) error {
+	requiredSA := resourceread.ReadServiceAccountV1OrDie(v410_00_assets.MustAsset("v4.1.0/kube-apiserver/localhost-recovery-sa.yaml"))
+	requiredToken := resourceread.ReadSecretV1OrDie(v410_00_assets.MustAsset("v4.1.0/kube-apiserver/localhost-recovery-token.yaml"))
+
+	saClient := client.ServiceAccounts(operatorclient.TargetNamespace)
+	serviceAccount, err := saClient.Get(requiredSA.Name, metav1.GetOptions{})
+	if err != nil {
+		if !apierrors.IsNotFound(err) {
+			return err
+		}
+
+		serviceAccount, err = saClient.Create(requiredSA)
+		if err != nil {
+			return err
+		}
+	}
+
+	// Create custom secret to be populated with SA token so we have a stable name
```

Comment:
don't understand that comment

Reply:
token secrets get random name, creating our own makes sure we have a stable name

Reply:
write that

Reply:
not sure I want to be the one mirroring kube docs but ok